akka-remote/src/main/scala/akka/remote/artery/compress/HeavyHitters.scala

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.remote.artery.compress
+
+import scala.collection.immutable
+
+// TODO describe / think about how we could handle an actor dying and removing from "top"
+// TODO this is the scenario when "super traffic actor" goes away and we want to not use up the slot in HH
```

Comment:
I imagined that we roughly use the following:
 1. Use table A, and in the meantime build new table B (from scratch!)
 2. Eventually start advertising B
 3. We see remote using B, switch to B and start from 1. (now B becoming A)